pyInterface/convertEvtToTree.py

```diff
@@ -105,7 +105,8 @@ def __str__(self):
 	parser.add_argument("outputFileName", help="The .root file to be written")
 	parser.add_argument("-u", "--userstring", help="User string", default="")
 	parser.add_argument("-t", "--type", dest="eventsTypeString", help="type of data (can be 'real', 'generated' or 'accepted', default: 'other')", default="other")
-	parser.add_argument("-b", "--binning", action='append', help="declare current bin in the form 'binningVariable;lowerBound;upperBound' (e.g. 'mass;1000;1100')."+
+	parser.add_argument("-b", "--binning", action='append',
+					                       help="declare current bin in the form 'binningVariable;lowerBound;upperBound' (e.g. 'mass;1000;1100')."+
```

Comment:
There are too many tabs in this indent. It should only be one tab, followed by enough spaces to align `help=` with `"-b",`.